R/mapDGEobj.R

```diff
@@ -1,29 +1,28 @@
 #' Construct DGEobj relationships
 #'
-#' Reads a DGEobj and either (1) produces a node pair file ready for plotting or (2) a plot visualizing parent/child relationships between
+#' Reads a DGEobj and visualizes parent/child relationships between
 #' data items in a DGEobj.
 #'
 #' @param dgeObj DGEobj to find the parent/child relationships between data items.
-#' @param plotType Must be canvasXpress or igraph (default = canvasXpress).
-#' @param directed Only for igraph. Indicates if the graph should
+#' @param plotType Must be canvasXpress or ggplot (default = canvasXpress).
+#' @param directed Only applies to ggplot. Indicates if the graph should
 #'     be directed or not. (default = TRUE)
 #'
-#' @return A class igraph network object or a canvasXpress network plot.
+#' @return A class ggplot object or a canvasXpress network plot.
 #'
 #'
 #' @examples
 #' \dontrun{
-#'   # Prepare canvasxpress network plot
+#'   # Prepare canvasXpress network plot
 #'   mynet <- mapDGEobj(dgeObj)
 #'
-#'   # Prepare an igraph object for plotting
-#'   mynet <- mapDGEobj(dgeObj, plotType = "ggplot")
-#'   plot(mynet)
+#'   if (requireNamespace("ggraph", quietly = TRUE) || requireNamespace("tidygraph", quietly = TRUE)) {
+#'       mynet <- mapDGEobj(dgeObj, plotType = "ggplot")
+#'     }
```

Comment:
This is in a don't run block, you can remove the requireNamespace Check.  We should assign the mynet and then also show the user how to plot it (ie via plot or print like in the old example.

Reply:
the function now returns a ggplot object so the user doesn't need to do any further modifications to see the plot so I removed the assignment to mynet all together